hacker_Rank/May14_Answer.py

```diff
@@ -0,0 +1,34 @@
+from algorithm import Algo
+
+obj = Algo()
+
+
+def solution(a):
+    print(a)
+    n = len(a)
+    sum = 0
+    master = []
+    for p in range(n):
+        for q in range(1, n - 1):
+            if 0 <= p < q < n:
+                print("P", p, "vs", q, "q")
+                for item in range(p, q + 1):
+                    # print("item" , item)
+                    sum = a[item] + sum
+                print("sum", sum)
+
+                avg = sum / (q - p + 1)
+                print("avg", avg)
```

Comment:
Program is Correct, 
refactor the code, 
create slicing pairs in a list using list comprehension, 
and then make use of that,
use ``` sum() ``` function,